functionnal_tests/cypress/integration/app_folder/translation.js

```diff
@@ -6,17 +6,17 @@ describe('App Folder Advanced', function () {
     cy.setupBaseDB()
     cy.loginAs('users')
     cy.fixture('baseWorkspace').as('workspace').then(workspace => {
-      cy.visit(`/ui/workspaces/${workspace.workspace_id}/contents?type=file`)
+      cy.visitPage({ pageName: PAGES.CONTENTS, params: { workspaceId: workspace.workspace_id } })
     })
   })
 
   it('should have translations', () => {
-    cy.get('.folder__header__name').contains('Received files')
+    cy.get('.workspace__header__title').contains('List of contents')
```

Comment:
Is that right? It doesn't look like App Folder Advanced to me.

Reply:
fixed this :-)